lib/gui/models/store.js

```diff
@@ -225,8 +226,8 @@ const storeReducer = (state, action) => {
         throw new Error('The drive is write-protected');
       }
 
-      // TODO: Reuse from SelectionStateModel.isDriveLargeEnough()
-      if (state.getIn([ 'selection', 'image', 'size' ], 0) > selectedDrive.get('size')) {
+      const image = state.getIn([ 'selection', 'image' ]);
```

Comment:
Tried it now, it results in `TypeError: Cannot read property 'toJS' of undefined` when `[ 'selection', 'image' ]` doesn't exits.